docs/SOPs/update_ena_runs_SOP.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: Update ENA runs SOP
+parent: SOPs
+---
+## Table of contents
+
+{: .no_toc .text-delta }
+
+1. TOC {:toc}
+## Overview
+This is the SOP for fixing datasets in the issue: ebi-ait/hca-ebi-wrangler-central#250
+
+## Requirements
+1. Gain access to the EBI cluster
+   ```
+   ssh ebi-cli.ebi.ac.uk
+   ```
+2. Install gsutil/awscli in your environment in the EBI cluster
```

Comment:
how?

Reply:
yup, i will add some instructions or link to instructions!

Reply:
done!